Answers.md

```diff
@@ -1,2 +1,28 @@
 # Your responses to the short answer questions should be laid out here using Mark Down.
 ### For help with markdown syntax [Go here](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)
+
+### 1. Describe some of the differences between `.forEach` & `.map`.
+#### Some of the differences between `.forEach` & `.map` are as follows:
+#### `.forEach` scours an array for a particular item in the array. Once it finds the index it allows us to make use of that item but it doesn't return that item the way `.map` would.
```

Comment:
`forEach` iterates through the entire array, not looking for a particular item, but rather applying every item in the array to a callback function that then does something to the items in the array. Does that make sense?